libs/dh/admin/feature-user-management/src/lib/user-management-tabs/users-tab/dh-users-tab-table.component.html

```diff
@@ -0,0 +1,41 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<watt-table
+  #table
+  [columns]="columns"
+  [dataSource]="dataSource"
+  sortBy="name"
+  sortDirection="asc"
+  [sortClear]="false"
+  [resolveHeader]="translateHeader"
```

Comment:
Nikpick: It is possible to use transloco directly as a function to resolveHeader

Reply:
Yes, I saw it used like that in another template. While possible, this means that the keys in the translation files must be the same as the column names. This works well sometimes, but in this case two of the column names have different values, so the custom function was necessary.